pkg/test/client.go

```diff
@@ -30,6 +30,8 @@ type frameworkClient struct {
 
 var _ FrameworkClient = &frameworkClient{}
 
+// FrameworkClient is a wrapper for the controller-runtime client with a modified Create function
```

Comment:
Here and everywhere else these should be full sentences (e.g. with a period at the end). See https://blog.golang.org/godoc-documenting-go-code
> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.

Reply:
Thanks. Didn't know godoc needed periods.